src/template/partials/footer.html

```diff
@@ -1,3 +1,95 @@
 <footer>
+  <div class="container">
+    <div
+      class="contact-container d-flex flex-column align-items-center justify-content-center"
+    >
+      <h1 class="contact-container--title">Ready to Get Started?</h1>
+      <p class="contact-container--info">
+        Data analysis software is a type of software tool used for data analysis
+        and reporting. It is designed to help businesses, organizations.
+      </p>
+      <a href="#" class="btn animated delay fadeIn"> Contact </a>
+    </div>
+  </div>
```

Comment:
you should move this outside of the footer tag, as it's not part of the footer, you can keep it in the component tho

Reply:
done.